src/app/[locale]/(auth)/login/page.tsx

```diff
@@ -11,8 +20,34 @@ export default function LoginPage() {
 
   return (
     <main className="h-full w-full bg-gradient">
-      로그인페이지
-      <button onClick={handleGoBack}>뒤로가기</button>
+      <div>
+        <p className="flex items-center justify-end p-3 pb-0">
```

Comment:
```suggestion
        <div className="flex items-center justify-end p-3 pb-0">
```

p 태그는 문단을 나타낼 때 쓰는 태그입니다.
영역을 잡기 위한 태그니 div나 span이 적절해 보입니다

Reply:
✅ 89876879a51ebbc1904c38000087abd7942eedda